app/grandchallenge/serving/models.py

```diff
@@ -118,11 +118,24 @@ def get_component_interface_values_for_user(
         .values_list("values__pk", flat=True)
     )
 
+    evaluation_query = filter_by_permission(
+        queryset=Evaluation.objects.all(),
+        user=user,
+        codename="change_evaluation",
```

Comment:
Unsure, but I think serving should only be allowed for challenge admins, not for participants. Participants should only be allowed to see the metrics that are exposed through the leaderboard (which might only be a subset in metrics.json), for example.

Reply:
Then again, the evaluation detail page is accessible to participants, and we print the entire metrics.json there, and also metrics.json is not a file type socket, so not actually an output to download anyway. Then by extension the same rights should apply to additional evaluation outputs, so maybe I should change the permission back to `view`.

Reply:
Thinking about whether to allow serving of evaluation inputs (might become necessary down the line if a challenge uses additional input of type file): that should definitely be restricted to admins then since the other inputs are the hidden test data.

Reply:
Could this be added as a comment in the code here for future developers?

Reply:
Sure, but what is your opinion on which permission to use for downloading the outputs?